pkg/api/v1/client/direct.go

```diff
@@ -69,7 +70,12 @@ func (c DirectScriptsAPI) GetScript(id string) (script testkube.Script, err erro
 }
 
 func (c DirectScriptsAPI) GetExecution(scriptID, executionID string) (execution testkube.Execution, err error) {
-	uri := c.getURI("/scripts/%s/executions/%s", scriptID, executionID)
+
+	uri := c.getURI("/executions/%s", executionID)
+
+	if scriptID != "" {
```

Comment:
remove this one handle only executions by ID

Reply:
✅